packages/@aws-cdk/aws-pipes-alpha/package.json

```diff
@@ -88,12 +88,14 @@
     "jest": "^29",
     "aws-cdk-lib": "0.0.0",
     "constructs": "^10.0.0",
-    "@aws-cdk/integ-tests-alpha": "0.0.0"
+    "@aws-cdk/integ-tests-alpha": "0.0.0",
+    "@aws-cdk/aws-kinesisfirehose-alpha": "0.0.0"
   },
   "dependencies": {},
   "peerDependencies": {
     "aws-cdk-lib": "^0.0.0",
-    "constructs": "^10.0.0"
+    "constructs": "^10.0.0",
+    "@aws-cdk/aws-kinesisfirehose-alpha": "0.0.0"
```

Comment:
Is the peer dependency required here?

Reply:
I think so.. because I am importing `IDeliveryStream` in `logs.ts`: 

`import { IDeliveryStream } from '@aws-cdk/aws-kinesisfirehose-alpha';` 

When I do `npx lerna run build --scope=@aws-cdk/aws-pipes-alpha` I get an error if I don't have it.